public/modules/custom/grants_webform_print/src/Controller/GrantsWebformPrintController.php

```diff
@@ -47,8 +47,10 @@ class GrantsWebformPrintController extends ControllerBase {
    * @param \Drupal\webform\WebformTranslationManager $translationManager
    *   Translation manager.
    */
-  public function __construct(LanguageManagerInterface $languageManager,
-                              WebformTranslationManager $translationManager) {
+  public function __construct(
+    LanguageManagerInterface $languageManager,
+    WebformTranslationManager $translationManager,
```

Comment:
These should use constructor promotion (https://stitcher.io/blog/constructor-promotion-in-php-8)